README.md

```diff
@@ -392,8 +395,13 @@ The `phase` property is set to 'audio-video' during the initial audio-video test
 secondary audio-only test is required (because audio quality was not acceptable during the
 audio-video test), the property is set to 'audio-only'.
 
+The `availableOutgoingBitrate` is calculated dynamically based on the current network conditions and the resources available on the sender's device. The sender can use this value to adjust the quality of the media stream, such as the resolution, frame rate, and bitrate of the video stream, to ensure that the receiver can receive the media with the best possible quality, while still maintaining a smooth and uninterrupted transmission.
```

Comment:
You write "The sender can use this value to adjust the quality of the media stream, such as the resolution, frame rate, and bitrate of the video stream." Can you give an example of how they would do that?

Reply:
Should the readme note the lack of support for `availableOutgoingBitrate` in Firefox (and its implications on the testQuallity results)?